nix/pkgs/docker-image/default.nix

```diff
@@ -5,6 +5,20 @@ let
 
     set -eu
 
+    # set defaults
+    amesPort=${toString amesPort}
+
+    # check args
+    for i in "$@"
+    do
+    case $i in
+        -p=*|--port=*)
+            amesPort="${i#*=}"
+            shift
+            ;;
```

Comment:
Per CI, this is a *nix* syntax error. Not sure how it should be escaped.

Reply:
Not sure, I tested this by updating the script directly in a container. Maybe @eamsden knows how?